stdlib/src/builtin/bool.mojo

```diff
@@ -245,3 +247,16 @@ struct Bool(Stringable, CollectionElement, Boolable, EqualityComparable):
             `value ^ self`.
         """
         return value ^ self
+
+    @always_inline("nodebug")
+    fn __int__(self) -> Int:
+        """Convert this Bool to an integer.
+
+        Returns:
+            1 if the Bool is True, 0 otherwise.
+        """
+        return __mlir_op.`pop.cast_to_builtin`[_type = __mlir_type.index](
+            __mlir_op.`pop.cast`[_type = __mlir_type.`!pop.scalar<index>`](
+                self.value
+            )
+        )
```

Comment:
There is a `Int` constructor from `!pop.scalar<index>`, so I think you could skip the `pop.cast_to_builtin`.

Reply:
You are right, I fixed it, thanks!